apps/vaccitron/src/app/login-page/fb-login/fb-auth/fb-auth.service.ts

```diff
@@ -0,0 +1,26 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { SocialAuthService, SocialUser } from 'angularx-social-login';
+import { environment } from '../../../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FbAuthService {
+  authEvent = new EventEmitter<SocialUser|null>();
```

Comment:
is this a way for the app to manage login events chosen by us ? or is a standard for the FB login ? 
if it is by us, aren't there are ways to save this state and subscriber to updates, other than events ?

Reply:
I'm going to redo it and see if using the FB sdk directly works. Something isn't working right in FF at least (didn't check Safari yet...)